CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+##2015-12-02 Supported Release 1.0.6
```

Comment:
Do we want the date in here?  All the other PRs had the date removed since it hasn't been shipped yet.